src/components/home/InformationPanel.vue

```diff
@@ -0,0 +1,44 @@
+<script lang="ts" setup>
```

Comment:
The collapsible behavior should be handled in another higher component, because it's shared to all pages.

Reply:
If we use pinia storage, isn't that enough since we use pinia to store the state?

Reply:
no, cause here you will have to duplicate the behavior for all pages